fn/chocolatey/functions.nuspec

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>functions</id>
+    <version>0.2.25</version>
+    <packageSourceUrl>https://github.com/iron-io/functions/</packageSourceUrl>
+    
+    <title>Iron.io Functions (Install)</title>
+    <authors>Iron.io Inc.</authors>
+    <projectUrl>https://github.com/iron-io/functions/</projectUrl>
```

Comment:
ok done. Removed both the trailing slash from the `<projectUrl>` and `<packageSourceUrl>`.